packages/node/package.json

```diff
@@ -36,6 +36,7 @@
     "merkle-tools": "^1.4.1",
     "parse-json": "^5.2.0",
     "pg": "^8.7.1",
+    "pino": "^6.11.3",
```

Comment:
pino is an indirect dependency via `@subql/common`, any reason we need to explicitly import it now?